src/main/java/core/screens/HUD/HudManager.java

```diff
@@ -0,0 +1,108 @@
+package core.screens.HUD;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.utils.Disposable;
+import com.badlogic.gdx.utils.viewport.FitViewport;
+import com.badlogic.gdx.utils.viewport.Viewport;
+
+public class HudManager implements Disposable {
+
+    public Stage stage;
+    private Viewport viewport;
+
+    //score && time tracking variables
+    private Integer worldTimer;
+    private float timeCount;
+    private static Integer score;
+    private boolean timeUp;
+
+    //Scene2D Widgets
+    private Label countdownLabel, timeLabel, linkLabel;
+    private static Label scoreLabel;
+
+    public HudManager(SpriteBatch sb) {
+        //define tracking variables
+        worldTimer = 250;
+        timeCount = 0;
+        score = 0;
+
+        //setup the HUD viewport using a new camera seperate from gamecam
+        //define stage using that viewport and games spritebatch
+        viewport = new FitViewport(Gdx.graphics.getWidth(), Gdx.graphics.getHeight(), new OrthographicCamera());
```

Comment:
i think it's possible & better to just get the camera and viewport from the existing CameraManager


if you set a renderer's projection matrix to (og gamecam) camera.combined, things are still drawn with respect to the camera (not the world) as expected

Reply:
I'll do this once I merge to staging and have access to the new cameraManager class.